components/carousel/Carousel.razor.cs

```diff
@@ -49,6 +49,24 @@ public partial class Carousel : AntDomComponentBase
 
         #endregion Parameters
 
+        public void Next() => GoTo(_slicks.IndexOf(_activeSlick) + 1);
+
+        public void Previous() => GoTo(_slicks.IndexOf(_activeSlick) - 1);
+
+        public void GoTo(int index)
+        {
+            if (index >= _slicks.Count)
+            {
+                index = 0;
+            }
+            else if (index < 0)
+            {
+                index = _slicks.Count - 1;
+            }
```

Comment:
Why is this logic? Should it be the other way around?

Reply:
It's wrap around logic akin to how the Bootstrap carousel works: https://getbootstrap.com/docs/4.0/components/carousel/
When index is past the end of the list, it will move to first slide, and when user is trying to move backwards past the first slide it will move to the N-1 slide (last). 

The idea is to have constant flow when use uses Next() Preview() if that's not what the user want they can use Goto(i) with extra logic to for example not go past the bounds of the list, etc.